src/templates/core/angular/getRequestBody.hbs

```diff
@@ -1,6 +1,6 @@
 export const getRequestBody = (options: ApiRequestOptions): unknown => {
 	if (options.body) {
-		if (options.mediaType?.includes('/json')) {
+		if (options.mediaType?.includes('application/json') || options.mediaType?.includes('+json')) {
```

Comment:
Why not make this just endsWith('json')?

Reply:
It may not always end with json, as there may be charset information in the header. JSON with always be application/json or something/something+json (but may be appended by "; charset=utf8")

Reply:
How about includes json then? Why not drop + and /?